lib/DDG/Spice/PackageTracking.pm

```diff
@@ -74,6 +74,43 @@ triggers query_nowhitespace => qr/^
                                 )
                                 $/xi;
 
+## CanadaPost
+triggers query_nowhitespace_nodash => qr/^
+                                (?:
+                                    [\d]{9,} |
+                                    [A-Z]{2}\d{9}CA)
+                                )
+                                $/xi;
+
+## DHL
+triggers query_nowhitespace_nodash => qr/^
+                                (?:
+                                    \d{9,} |
+                                    \d{10} |
```

Comment:
Looking online, I found a couple other formats that we should support:

- https://www.trackingex.com/dhl-global-mail-asia-tracking.html
- https://www.trackingex.com/dhlbenelux-tracking.html